creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -22,7 +22,9 @@ const handleSyncFromPrimary = async ({
   forceResyncConfig,
   forceWipe,
   logContext,
-  blockNumber = null
+  secondarySyncFromPrimaryLogger,
+  blockNumber = null,
+  start = Date.now()
```

Comment:
weird to pass this in just for a `logger.info(beginnodesync)` - can just do that in the calling function and remove this param?

Reply:
youre right. removing